src/renderer/lib/migrations.js

```diff
@@ -43,10 +43,23 @@ function run (state) {
     migrate_0_17_0(state.saved)
   }
 
+  if (semver.lt(version, config.APP_VERSION)) {
+    installHandlers(state.saved)
+  }
+
   // Config is now on the new version
   state.saved.version = config.APP_VERSION
 }
 
+// Whenever the app is updated,  re-install default handlers if the user has
+// enabled them.
+function installHandlers (saved) {
+  if (saved.prefs.isFileHandler) {
+    const handlers = require('../../main/handlers')
```

Comment:
I don't think you can do this from the renderer process.

You have to use `ipcRenderer` to tell the main process to reinstall the handlers.